src/pages/wishlist/index.tsx

```diff
@@ -15,12 +15,12 @@ const Wishlist = () => {
   if (isLoading) return <LoadingPrimary />;
 
   return (
-    <div>
+    <Box>
       <Typography mb={{ xs: 4, md: 6, lg: 10 }} variant={'heading'}>
         {t(WishlistTypes.title)}
       </Typography>
 
-      <Box sx={mainStyles.accommmodationCard}>
+      <Box sx={{ ...mainStyles.accommmodationCard, justifyContent: 'flex-start' }}>
         {wishlist?.length ? (
```

Comment:
why do you need this check? you can do just `{wishlist?.map(...)}` 
in case the wishlist isn't present, it won't render

Reply:
wishlist can return as empty array